src/components/RadioPanel/index.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import { RadioPanel } from '.'
+import styled from '@emotion/styled'
+import { color, space } from '../../theme'
+import { Text } from '../Text'
+import { Checkmark } from '../../icons'
+
+export default {
+  title: 'Components/Inputs/RadioPanel',
+}
+
+const Price = styled(Text)`
+  color: ${color.foregroundMuted};
+  margin-inline-start: ${space[40]};
+`
+
+const USPS = styled.div`
+  margin-block-start: ${space[16]};
+  margin-block-end: ${space[16]};
+`
+
+const USP = styled.div`
+  display: flex;
+  gap: ${space[8]};
+  margin-block-end: ${space[16]};
+`
+
+export const Basic = () => {
+  const [value, setValue] = useState('')
+  const [loading, setLoading] = useState('')
+
+  return (
+    <RadioPanel
+      name="Refund Protection"
```

Comment:
Let's try to make these stories a bit less TS specific 😝

Reply:
Changed 🤪